최창익/app/src/main/java/com/mash_up/mvvmstudy/BindingAdapter.kt

```diff
@@ -0,0 +1,17 @@
+package com.mash_up.mvvmstudy
+
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import androidx.databinding.BindingAdapter
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+
+@BindingAdapter("app:imageUrl")
+fun loadImage(imageView: ImageView, url: String) {
+    Glide.with(imageView.context)
+        .load(url)
+        .placeholder(ContextCompat.getDrawable(imageView.context, R.drawable.mashup))
+        .error(ContextCompat.getDrawable(imageView.context, R.drawable.mashup))
+        .apply(RequestOptions().fitCenter())
+        .into(imageView)
```

Comment:
error 이미지까지 정의해주시고 있어서 bindingAdapter에서 errorImageRes, placeHolderRes까지 받아도 좋겟네요!

```suggestion
@BindingAdapter(value=["app:imageUrl", "error"], requireAll = false)
fun loadImage(
    imageView: ImageView, 
    url: String,
    @DrawableRes placeHolderRes: Int = R.drawable.mashup
    @DrawableRes errorDrawableRes: Int = R.drawable.mashup
) {
    Glide.with(imageView.context)
        .load(url)
        .placeholder(ContextCompat.getDrawable(imageView.context, placeHolderRes))
        .error(ContextCompat.getDrawable(imageView.context, errorDrawableRes))
        .apply(RequestOptions().fitCenter())
        .into(imageView)
```

Reply:
크.. errorImageRes, placeHolderRes 받는게 훨씬 낫겠군요. 감사합니다